root/edit/search_macros.tt

```diff
@@ -356,7 +357,7 @@
          predicate_subscription(linked_type);
        END %]
 
-    [% FOR linked_type=[ 'area', 'instrument', 'place', 'recording', 'release_group', 'release', 'work', 'editor' ];
+    [% FOR linked_type=[ 'area', 'event', 'instrument', 'place', 'recording', 'release_group', 'release', 'work', 'editor' ];
```

Comment:
This should of course be based on `%ENTITIES` at some point, too; perhaps when it is converted to react.